dvc/repo/run.py

```diff
@@ -109,7 +99,7 @@ def run(self, fname=None, no_exec=False, single_stage=False, **kwargs):
         if not is_valid_name(stage_name):
             raise InvalidStageName
 
-    params = parse_params(kwargs.pop("params", []))
+    params = chunk_dict(parse_params_from_cli(kwargs.pop("params", [])))
```

Comment:
`create_stage` still takes a list of params sadly so we have to chunk here (this will be fixed if/when we remove support for `--single-stage`).